py/dml/messages.py

```diff
@@ -1639,6 +1658,13 @@ class ELOGGROUPS(DMLError):
     fmt = ("Too many loggroup declarations. A maximum of 63 log groups (61 "
            + "excluding builtins) may be declared per device.")
 
+
+class ENOPROV(DMLError):
+    """
+    An invalid identifier was passed in the `provisional` statement.
+    """
+    fmt = "No such provisional feature %s. Valid values are: %s"
```

Comment:
I'd rather direct people to `--help-provisional`... once we've added that.

Reply:
I don't think we should add `--help-provisional`: provisionals are enabled only by syntax and thereby technically a part of the language, and describing the language is out of scope for help flags.

We may need to introduce flag-enabled provisionals later, but I hope not.

Reply:
Discussed outside R.C.: he does not, and there are some good arguments for not liking it. How we'll approach device-wide provisionals will have to be resolved later.